local-cli/bundle.js

```diff
@@ -13,18 +14,19 @@ function getBundle(flags) {
   var outPath = flags.out ? flags.out : OUT_PATH;
 
   var projectRoots = [path.resolve(__dirname, '../../..')];
-  if (flags.root) {
-    projectRoots.push(path.resolve(flags.root));
+  if (flags.roots) {
```

Comment:
+1 to a deprecation message. Whether it just warns and fails, or warns and makes `root` work for now, either sound good to me.